stglib/core/utils.py

```diff
@@ -1405,7 +1405,11 @@ def create_water_level_var(ds):
     Create water level variable from NAVD88 sensor height
     """
 
-    if ds.z.attrs["geopotential_datum_name"] == "NAVD88":
+    if "P_1ac" not in list(ds.data_vars):
+        print("Cannot create water_level variable without P_1ac")
```

Comment:
If this is an error state you should raise the appropriate error (maybe ValueError?)

Reply:
I don't want to throw an error here.  I still want the code to finish and produce the netcdf file.  This is just a message that will pop up if you are processing without doing the atmospheric correction to let you know there won't be a water_level variable without P_1ac.

Reply:
Okay. In that case, why is there an exit() call?

Reply:
I changed this to: if "P_1ac" in list(ds.data_vars) and ds.z.attrs["geopotential_datum_name"] == "NAVD88":